src/class-fields/double-empty-var.case

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Empty var, empty var
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  a; b;
+
+//- testfield
+  isEqualTo(this, "a", undefined);
+  verifyEnumerable(this, "a");
+  verifyWritable(this, "a");
+  verifyConfigurable(this, "a");
+
+  isEqualTo(this, "b", undefined);
+  verifyEnumerable(this, "b");
+  verifyWritable(this, "b");
+  verifyConfigurable(this, "b");
```

Comment:
yes, I have plans to improve verifyProperty for completeness. I'm lacking on time. For now, it's good for asserting enumerability, configurability and writability.